syz-manager/manager.go

```diff
@@ -544,6 +554,11 @@ func (mgr *Manager) runInstance(index int) (*Crash, error) {
 		return nil, fmt.Errorf("failed to setup port forwarding: %v", err)
 	}
 
+	_, err = inst.Copy(mgr.pcsBitmapPath)
+	if err != nil {
```

Comment:
Isn't coverage filter optional? And we copy it unconditionally. Shouldn't here be some if to check if the filter is enabled or not?